### ISSUE TITLE
Bower version pointing to another respository

### ISSUE THREAD

**Issue by ifthenelse** [constancecchen/object-fit-polyfill#7]
Doing a Bower download with `bower install object-fit-polyfill.js` retrieves an [older fork](https://github.com/db2k/object-fit-polyfill.git).

Shouldn't it point to this very repo, since it's the original (and hopefully most updated) version?

Also there is a project with a similar name: `bower install object-fit-polyfill` which is currently an apparently unmantained, 1 year old repo and points to https://github.com/tonipinel/object-fit-polyfill.

Can you provide clearer instructions for those still using Bower?

**Comment by cee-chen** [comment]
Hey hey, sorry for the delayed response! I was out of the country this last week.

Hmm, it looks like @db2k registered his fork on bower. I have no idea how to transfer ownership, so I'm going to go ahead and register it as a new package called `objectFitPolyfill`. Looks like there's a bunch of different `object-fit-*` packages, which is probably to be expected :)

I've poked around both bower and npm (sorry, I'm a bit of a package publishing noob), and I should have registered/published this now! I've added the commands to the README, but here they are as well:

```
bower install objectFitPolyfill
```

```
npm install objectFitPolyfill
```

Hope that helps, and thanks for letting me know about this!
